clinica/iotools/converters/adni_to_bids/adni_modalities/adni_av45_fbb_pet.py

```diff
@@ -10,6 +10,7 @@ def convert_adni_av45_fbb_pet(
     conversion_dir: PathLike,
     subjects: Optional[List[str]] = None,
     mod_to_update: bool = False,
+    n_procs: Optional[int] = 1,
```

Comment:
Optional can be dropped now.

```suggestion
    n_procs: int = 1,
```